src/styles/variables/_colors.scss

```diff
@@ -0,0 +1,42 @@
+/* main colors */
+$brandColor: #113f6d;
+$zodiac: #122844;
+$rhino: #273b54;
+$fiord: #3f5167;
+$shuttle: #59697d;
+$regent: #818b99;
+$mischka: #d5dae0;
+$turquoise: #59d2ef;
+$turquoiseLowAlpha: rgba($turquoise, 0.15);
+$foam: #e0f6fd;
+$pacific: #007e9b;
+$concrete: #f2f2f2;
+$nurse: #e9ebe7;
+
```

Comment:
Je ne suis pas fan de mettre le nom de la couleur dans la variable, car demain si on change le theme, le turquoise sera zinzolin ? non sens :). Et si on doit modifier partout le nom de la variable, cela n'a pas d'interet de faire un fichier de variable ;)

Reply:
I'm not disappointed by this practice that much : as we distinguish the names of the colors of our palette from how they are used in the project. It kinda takes part in the expression of the brand's identity :)